### ISSUE TITLE
[Feature]Settings & search box to be visible during scroll (like sticky)

### ISSUE THREAD

**Issue by vdbhb59** [marienfressinaud/FreshRSS#5680]
**Is your feature request related to a problem? Please describe.**
When one is scrolling the feeds, you will notice the settings icon & the search box, and even the name FreshRSS becomes invisible. They are not sticky like the function box is (show read/show unread/.../refresh/.. etc). (See snap # 4)

**Describe the solution you’d like**
That particular header or the margin header should be sticky throughout, to make it simpler for anyone to click and navigate to settings or search anytime. (See snap # 3rd onward)

**Describe alternatives you’ve considered**
I have to scroll up many pages, which makes it very tiresome. The "navigation" is also not visible, or not visible all the time (settings > display > show navigation). I tried every theme, and it is the same. The navigation bar starts to load, but then vanishes upon the full page loading. (See 1st 2 snaps at the bottom).

**Additional context**
All snaps, that I can manage to share are attached here... I have around 1030+ users right now, and many have come back to me asking is that something available in FreshRSS? Trouble is, I tried different browsers as well, without ad-blockers, with ad-blockers, with or without java/script/..any kind of blockers, but the navigation menu vanishes upon loading. I checked in different phone and different browsers there too, but it is same. I checked with another person who is running his own instance of FreshRSS (personal), and he has the same issue. Both of us are hosting on separate architectures, separate OSs, separate systems, everything different, but this is same. Nav bar vanishes.
I have given the context of nav bar so to let developers know I have checked that option/function, and it is not working. This same nav bar was pulled up in a different issue for the FreshRSS homepage issue under https://github.com/FreshRSS/freshrss.org/issues/59
My aim here is to have the search and settings gear to be present as a sticky like the actions bar.

![image](https://github.com/FreshRSS/FreshRSS/assets/60728004/cb39b9ad-e71f-49f7-b86b-62971719a8a5)
![image](https://github.com/FreshRSS/FreshRSS/assets/60728004/907aadc2-9843-4065-a29e-241af38988de)
![image](https://github.com/FreshRSS/FreshRSS/assets/60728004/a808f81e-2817-422f-a03d-33dace3e5c25)
![image](https://github.com/FreshRSS/FreshRSS/assets/60728004/8d6e66c1-85ab-44b1-a51d-c9164905ce90)

**Comment by Frenzie** [comment]
> I have to scroll up many pages, which makes it very tiresome. The "navigation" is also not visible, or not visible all the time (settings > display > show navigation). I tried every theme, and it is the same. The navigation bar starts to load, but then vanishes upon the full page loading. (See 1st 2 snaps at the bottom).

That's not normal. Could you try without extensions?

**Comment by vdbhb59** [comment]
> > I have to scroll up many pages, which makes it very tiresome. The "navigation" is also not visible, or not visible all the time (settings > display > show navigation). I tried every theme, and it is the same. The navigation bar starts to load, but then vanishes upon the full page loading. (See 1st 2 snaps at the bottom).
> 
> That's not normal. Could you try without extensions?

Okay. I found it was due to:

Add touch gestures to FreshRSS. — By: Marco Heizmann < this extension.

However, my point remains: the settings gear like the sticky menu should be available along with the search bar at the top. That feature should be available, it is valuable.

**Comment by Frenzie** [comment]
I'd rather simply show the header on scroll up instead of adding all kinds of distracting UI elements. :)

**Comment by vdbhb59** [comment]
> I'd rather simply show the header on scroll up instead of adding all kinds of distracting UI elements. :)

Agreed. That works too. Settings gear is something which can be made visible. It is an important aspect.

**Comment by math-GH** [comment]
Just for the record: it depends a bit of the selected theme. The `swage` theme has a sticky header

**Comment by vdbhb59** [comment]
So, would this be added as a feature or like how?

**Comment by math-GH** [comment]
> So, would this be added as a feature or like how?

Would it be ok to have an extension first for that feature?
For the long run I will overthink the current situation to improve the UI/UX in general. But step by step ;)

**Comment by vdbhb59** [comment]
Fair enough. Just wanted to know if this is something which can be done or not. : -)

**Comment by math-GH** [comment]
This will be part of #5966
